torch/csrc/jit/codegen/cuda/lower2device.cpp

```diff
@@ -261,6 +261,16 @@ class GpuLower::KernelIrMapper : private OptInConstDispatch {
   }
 
   void handle(const ReductionOp* node) final {
+    // If trivial reduction operation lower to set operation.
```

Comment:
It's cool that we are starting to convert trivial `ReductionOps` to `SET`. We might want to clean up `initReduction` in `lower_loops` pass as well, saving a redundant loop from the generated kernel. 

https://github.com/csarofeen/pytorch/blob/20_12_3_devel/torch/csrc/jit/codegen/cuda/lower_loops.cpp#L180-L190

Reply:
Good catch, added this logic.